arch/Kconfig

```diff
@@ -199,7 +199,7 @@ config PRIVILEGED_STACK_TEXT_AREA
 
 config KOBJECT_TEXT_AREA
 	int "Size if kobject text area"
-	default 512 if COVERAGE_GCOV
+	default 2048 if COVERAGE_GCOV
```

Comment:
Master should not need this change anymore.

Reply:
yes, I see your PR, thanks